extensions/wikia/ArticleComments/classes/ArticleComment.class.php

```diff
@@ -197,84 +192,124 @@ static public function newFromId( $id ) {
 	 * @return bool
 	 */
 	public function load( $master = false ) {
-		wfProfileIn( __METHOD__ );
-		$result = true;
+		if ( $this->isLoaded ) {
+			return true;
+		}
 
-		if ( $this->mTitle ) {
- 			// get revision ids
```

Comment:
All of this convoluted code was moved to `loadFirstRevId` and `loadLastRevId` methods.